libraries/chain/include/graphene/chain/protocol/transaction.hpp

```diff
@@ -163,15 +163,17 @@ namespace graphene { namespace chain {
        * @param get_owner  callback function to retrieve owner authorities of a given account
        * @param allow_non_immediate_owner whether to allow owner authority of non-immediately
        *            required accounts to authorize operations in the transaction
+       * @param ignore_custom_operation_required_auths See issue #210; whether to ignore the
+       *            required_auths field of custom_operation or not
        * @param max_recursion maximum level of recursion when verifying, since an account
        *            can have another account in active authorities and/or owner authorities
        */
-      void verify_authority(
-         const chain_id_type& chain_id,
-         const std::function<const authority*(account_id_type)>& get_active,
-         const std::function<const authority*(account_id_type)>& get_owner,
-         bool allow_non_immediate_owner,
-         uint32_t max_recursion = GRAPHENE_MAX_SIG_CHECK_DEPTH )const;
+      void verify_authority(const chain_id_type& chain_id,
+                            const std::function<const authority*(account_id_type)>& get_active,
+                            const std::function<const authority*(account_id_type)>& get_owner,
+                            bool allow_non_immediate_owner,
+                            bool ignore_custom_operation_required_auths = false,
```

Comment:
Please remove the default value. We don't want to miss invocations that need to be adapted.